packages/loader/container-loader/src/prefetchDocumentStorageService.ts

```diff
@@ -2,6 +2,9 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
+
+/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
I'm not big fan of this, I'd rather keep things as is, as it increases the bar for new violations to be added

Reply:
Reverted